CHANGELOG.md

```diff
@@ -139,6 +139,14 @@
 - **Zipkin**: add support for including HTTP path in span name
   through configuration property `http_span_name`.
   [#8150](https://github.com/Kong/kong/pull/8150)
+- **Zipkin**: add support for socket connect and send/rev timeouts
```

Comment:
I think the timeout options were in another PR. Could you rebase this PR on the master branch?

Reply:
This is branched from #8735 (still in review) because it depends on the new tests added there. I'd normally change the base branch in GitHub so that only the new commits show up in the diff but I'm not able to do that across two forks, so instead I'll rebase when #8735 is merged.